.github/workflows/main.yml

```diff
@@ -30,8 +34,14 @@ jobs:
       - uses: actions/setup-python@e797f83bcb11b83ae66e0230d6156d7c80228e7c # v6.0.0
         with:
           python-version: ${{ matrix.python-version }}
+      - name: Downgrade pip for riot compatibility
```

Comment:
```suggestion
```

maybe try without this too

Reply:
this was needed, [c74847e](https://github.com/DataDog/dd-apm-test-agent/pull/255/commits/c74847ed1838c0bb2a949a6c0be88a597454f587) failed removing everything but adding just this part back in [d9187ef](https://github.com/DataDog/dd-apm-test-agent/pull/255/commits/d9187ef33965005cf09869a52f7584029a943494) made everything pass